src/time-picker/tests/functional/TimePicker.ts

```diff
@@ -12,7 +12,7 @@ import * as textinputCss from '../../../theme/text-input.m.css';
 import { uuid } from '@dojo/framework/core/util';
 
 const axe = services.axe;
-const DELAY = 300;
+const DELAY = 500;
```

Comment:
Fixed delays should always be a last resort since they add up and slow down test suites. Is there some condition that can be polled for?

Reply:
The test that was failing specifically was `disabled menu items cannot be clicked` specifically the lines: 
```
			.then((isEqual) => {
				assert.isTrue(isEqual);
})
```
It was only failing during the pipeline, not during local testing, which made it difficult to debug. Bumping up the timing was a last resort, I didn't know what else could be making the travis pipeline fail but not on local.

Reply:
What I can do it bump up the delay time for this test only.